### ISSUE TITLE
Create develop branch in `meta-zarhus`

### ISSUE THREAD

**Issue by DaniilKl** [zarhus/zarhus-issues#20]
We should create `develop` branch so not to merge to `main` directly. Maybe this should be solved during https://github.com/zarhus/docs/pull/5.

**Comment by tym2k1** [comment]
[`develop` branch added](https://github.com/zarhus/meta-zarhus/tree/develop)

**Comment by DaniilKl** [comment]
@tym2k, have you restricted pushing directly into `main`?

**Comment by tym2k1** [comment]
No, I don't have such access rights. Someone with access should then additionally restrict pushing to `main` and accepting PR's that target `main`. Perhaps @macpijan?

**Comment by m-iwanicki** [comment]
@DaniilKl can this issue be closed?
